client/js/stores/day-store.js

```diff
@@ -1,9 +1,36 @@
 var Dispatcher = require('./dispatcher/dispatcher');
 var AppConstants = require('./constants/constants');
 var EventEmitter = require('event').EventEmitter;
+var assign = require('object-assign');
+var CHANGE_EVENT = 'change';
```

Comment:
glad we discussed this! we'll move it into the constants file if we deem it necessary